documentation/external-resources.md

```diff
@@ -0,0 +1,18 @@
+# [Documentation](./README.md) / External Resources
+
+This is a list of articles and talks that have influenced the advice in
+this section; you may find it useful to read or view these if you're
+especially interested in this topic.
+
+## Articles
+
+* [10 Tips For Making Your Documentation Crystal Clear](https://opensource.com/life/16/11/tips-for-clear-documentation), Ben Cotton
+* [A Primer on Documentation Content Strategy](https://increment.com/documentation/primer-on-documentation-content-strategy/), Stephanie Blotner
+* [What to Write](https://jacobian.org/writing/what-to-write/) and [Technical Style](https://jacobian.org/writing/technical-style/), Jacob Kaplan-Moss
```

Comment:
^He wrote the Django documentation I believe?  I know Django was also born out of a Newspaper CMS so that's not surprising they are writing experts.

Reply:
Jacob will be proud to know he's a writing expert. Django was born down the street from me in Lawrence, for the newspaper we read every day.

Reply:
Well, I can't speak for anyone else, but I found his articles pretty helpful at least. ;)